### PR TITLE
[fix] upgrade port settings

### PR DESCRIPTION
**Pull request by nemsia**

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -18,6 +18,13 @@ source /usr/share/yunohost/helpers
 domain=$(ynh_app_setting_get "$app" domain)
 path=$(ynh_app_setting_get "$app" path)
 is_public=$(ynh_app_setting_get "$app" is_public)
+port=$(ynh_app_setting_get "$app" port)
+
+# test port settings
+if [[ "$port" = "" ]]; then
```
Comment (JimboJoe) [inline scripts/upgrade]: `if [ -z "$port"] ; then` would be much better

**Review by JimboJoe — CHANGES_REQUESTED** [review]
Could you please also add `systemctl restart grafana-server` on line 34? That's not strictly mandatory, but that seems logical and it prevents package_check from failing.
The idea is to maintain level 7 for this app ;-)

File: scripts/upgrade (hunk 1 above)
Comment (nemsia) [reply scripts/upgrade]: Fix on https://github.com/YunoHost-Apps/grafana_ynh/pull/8/commits/7b610bf02e0fdc2116733ed907602cc38056a20f

**Comment by nemsia** [comment]
* Add restart https://github.com/YunoHost-Apps/grafana_ynh/pull/8/commits/4f453504891b30a2bf0ad8dc9dd71963b451e10c
* Better port check https://github.com/YunoHost-Apps/grafana_ynh/pull/8/commits/7b610bf02e0fdc2116733ed907602cc38056a20f
Tested on my own test server.

**Review by JimboJoe — APPROVED** [review]
Thanks again!

**Comment by JimboJoe** [comment]
@nemsia Thanks! Don't forget next time to work in a branch directly in the Yunohost-Apps repository; I will then be able to participate to the PR ;-)

**Pull request merged**